docs/source/configuration.rst

```diff
@@ -650,6 +650,48 @@ Developer configuration options
     Time in seconds to wait after persistent volume gets initialized to prevent
     mount errors on BLAST kubernetes jobs.
 
+.. _elb_disable_janitor:
+
+``ELB_DISABLE_AUTO_SHUTDOWN``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    **Set via an environment variable**.
+
+    * Default: Disabled
+    * Values: Any string. Set to any value to enable.
+
+    Disables the auto-shutdown feature.
+
+.. _elb_janitor_schedule:
+
+``ELB_JANITOR_SCHEDULE``
```

Comment:
Should we expose this one to users?

Reply:
This is not really an end-user configuration parameter, hence its location in the developer configuration options section.